.github/workflows/ci.yml

```
@@ -128,13 +128,174 @@ jobs:
+          retention-days: ${{ github.repository == 'NVIDIA/warp' && 7 || 1 }}
+
+  test-windows-import-libraries:
+    if: github.event_name != 'schedule' || github.repository == 'NVIDIA/warp'
+    name: test-windows-import-libraries (${{ matrix.platform }})
+    needs: build-windows-wheels
+    runs-on: ${{ matrix.os }}
+    permissions:
+      contents: read
+    env:
+      UV_PYTHON: ${{ matrix.python_request || '3.12' }}
+      WARP_CACHE_PATH: ${{ runner.temp }}/warp-cache-${{ matrix.platform }}-wheel
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)